StudentTrackingSystem/StudentTrackingSystemApp/views.py

```diff
@@ -165,7 +165,9 @@ def get_student_data_api(request):
     from django.core import serializers
     from datamodel.models import Student
 
-    serializedData = serializers.serialize("json", Student.objects.all())
+    lastEntry = Student.objects.last().upload_set
```

Comment:
instead of going through the Students table, getting the last student, and pulling its upload set, would it be simpler to just get the last entry from the UploadSet table?